src/AerosolModel.jl

```diff
@@ -0,0 +1,64 @@
+"""
+    A container for information on aerosol size distribution
+    and chemical properties.
+
+    The size distribution is a sum of lognormal internally mixed modes.
+"""
+module AerosolModel
+
+export Mode
+export AerosolDistribution
+
+"""
+    Mode
+
+Represents the sizes and chemical composition
+of aerosol particles in one size distribution mode.
+The mode is assumed to be made up of internally mixed components
+and follow a lognormal size distribution.
+"""
+struct Mode{T}
+    "geometric mean dry radius"
+    r_dry::Real
+    "geometric standard deviation"
+    stdev::Real
+    "total number concentration"
+    N::Real
```

Comment:
We'll need these types to be concrete to reside on the GPU, so maybe just defined with `Mode{T, FT}`? I believe that `Int64` below is concrete, so that should be fine.

Reply:
Are we testing CloudMicrophysics.jl on GPUs? Would be a useful thing to add. Just to be able to check if the functions can be called.

Reply:
done